classes/class-twenty-twenty-one-customize.php

```diff
@@ -81,6 +81,38 @@ public function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Add excerpt or full text selector to customizer
+			 */
+			$wp_customize->add_section(
+				'theme_settings',
+				array(
+					'title'    => esc_html__( 'Theme settings', 'twentytwentyone' ),
+					'priority' => 30,
+				)
+			);
+
+			$wp_customize->add_setting(
```

Comment:
The setting needs to have a sanitize_callback. For the radio option, we need to validate the data.

A new function needs to be added, but you can refer to this example: 
https://github.com/WPTT/code-examples/blob/master/customizer/sanitization-callbacks.php#L278